src/cmd/MVP-1/main.go

```diff
@@ -1,8 +1,8 @@
 package main
```

Comment:
we shouldn't be editing the MVP-1 main when we changes change the code, the MVP-1 was a previous varsion and all of this is for the next MVP